lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -292,17 +309,70 @@ instance ToText Log where
             toText l
         MsgAccountNotFound account ->
             "Reward account not found: " <> pretty account
-        MsgDiag t -> t
+        MsgGetAddressTxs f t a ->
+            T.unwords
+                [ "Getting transactions from"
+                , pretty f
+                , "till"
+                , pretty t
+                , "for the"
+                , either
+                    (("address " <>) . pretty)
+                    (("reward account " <>) . pretty)
+                    a
+                ]
+        MsgFetchTransaction h ->
+            "Fetching transaction: " <> pretty (BF.unTxHash h)
+        MsgFetchDelegation h ->
+            "Fetching delegation: " <> pretty (BF.unTxHash h)
+        MsgFetchedLatestBlockHeader bh ->
+            "Fetched latest block header: " <> pretty bh
+        MsgCurrentSlottingParameters ->
+            "Fetching current slotting parameters..."
+        MsgEraByLatestEpoch era epoch ->
+            T.unwords
+                [ "Latest Cardano era is"
+                , T.pack $ show era
+                , ", determined by the latest epoch"
+                , pretty epoch
+                ]
+        MsgFetchNetworkRewardAccountBalances ->
+            "Fetching network reward account balances..."
+        MsgIsConsensus cp b ->
+            T.unwords
+                [ "ChainPoint"
+                , pretty cp
+                , if b then "does" else "doesn't"
+                , "belong to consensus"
+                ]
+        MsgBlockHeaderAtHeight height mbh ->
+            "Fetched BlockHeader at height" <> pretty height
+                <> ": " <> pretty mbh
+        MsgBlockHeaderAt cp mbh ->
+            "Fetched BlockHeader at " <> pretty cp
+                <> ": " <> pretty mbh
+        MsgGotNextBlocks n ->
+            "Fetched " <> pretty n <> " next blocks"
 
 instance HasSeverityAnnotation Log where
     getSeverityAnnotation = \case
-        MsgTipReceived _ -> Info
-        MsgTipWatcherNotified _ -> Debug
+        MsgTipReceived {} -> Info
+        MsgTipWatcherNotified {} -> Debug
```

Comment:
```suggestion
        MsgTipReceived{} -> Info
        MsgTipWatcherNotified{} -> Debug
```

Style: I prefer to put record braces immediately after the constructor — this preserves the style property that I can distinguish function arguments by looking at whitespace (otherwise, I may visually parse `{}` as a separate argument to `MsgTipWatcherNotified`, which it is not).

Same for the other cases.

Reply:
Yes, I understand your point. I have no problem with doing it your way if it helps you to interpret expressions right. Personally, I am ok both ways.